<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>830</width>
    <height>540</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="btnSelectPath">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>510</y>
     <width>120</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Selecionar Diretório</string>
   </property>
   <property name="icon">
    <iconset>
     <normalon>images/folder--arrow.png</normalon>
    </iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="btnDeleteRow">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>483</y>
     <width>100</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Elimina Linha(s)</string>
   </property>
   <property name="icon">
    <iconset>
     <normalon>images/cross.png</normalon>
    </iconset>
   </property>
  </widget>
  <widget class="QScrollArea" name="saCheckBoxList">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>20</y>
     <width>340</width>
     <height>460</height>
    </rect>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>336</width>
      <height>456</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="lblExtracao">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Resultado da Extração</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnProcessar">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>510</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Processar</string>
   </property>
   <property name="icon">
    <iconset>
     <normalon>images/gear.png</normalon>
    </iconset>
   </property>
  </widget>
  <widget class="QRadioButton" name="rb_layout">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>485</y>
     <width>82</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Obter Layout</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btSave">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>510</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Salvar</string>
   </property>
   <property name="icon">
    <iconset>
     <normalon>images/disk.png</normalon>
    </iconset>
   </property>
  </widget>
  <widget class="QLabel" name="lblTextoLido">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>0</y>
     <width>100</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Arquivos Disponíveis</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSair">
   <property name="geometry">
    <rect>
     <x>745</x>
     <y>510</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Sair</string>
   </property>
   <property name="icon">
    <iconset>
     <normalon>images/door-open-out.png</normalon>
    </iconset>
   </property>
  </widget>
  <widget class="QRadioButton" name="rb_Dados">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>485</y>
     <width>82</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Obter Dados</string>
   </property>
  </widget>
  <widget class="QTableView" name="tbvExtratacao">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>460</width>
     <height>460</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
